### ISSUE TITLE
Dailyrain clearing from DB too early

### ISSUE THREAD

**Issue by maxwellpower** [acuparse/acuparse#8]
Seems dailyrain entries are getting cleared before midnight. Appears related to the way Acurite sends the time back and the readings are clearing too early(Issue #6). Wiping all the data from the previous day.

Need to add checks to prevent zeroing out dailyrain before midnight server time.

![2017-07-01 11_23_05-www ghwx ca _ localhost _ acuparse _ dailyrain _ phpmyadmin 4 2 12deb2 deb8u2](https://user-images.githubusercontent.com/20729211/27764167-ef26a08a-5e4f-11e7-95ea-6a82cb527241.png)

**Comment by maxwellpower** [comment]
Seems Acurite is sending localtime responses with a time greater than system time. Which resets the rain counter for the day. Modified GHWX, forcing the server to send system time as the response to the hub instead of whatever Acurite is sending back. Will monitor and see if that clears up the issue.

**Comment by maxwellpower** [comment]
So it would seem I was on the right track with this daily rain issue originally but messed up the implementation pretty good.

Acurite is sending the time back as H:H:H which does clear the rain counters as expected but is still a little weird. I couldn't understand why the hub would not clear the counters when sent H:i:s instead.

Turns out, the regex I used to initially solve the issue was bad. I had '/["localtime":"](\d{2}:\d{2}:\d{2})/' but that was returning the result "localtime":H:H:H" which is missing a quotation. The proper format is "localtime":"H:H:H". This caused the hub to ignore the localtime response and not reset the rain counters at all.

Correcting the regex to '/(\d{2}:\d{2}:\d{2})/' and Acuparse is now returning the properly formatted JSON, using the current time. After some testing, this appears to now be clearing the rain counters as expected.

I'm still puzzled why Acurite sends back H:H:H. I really don't think it matters all that much. Since the hub counters only seem to rely on the first H value, when resetting. Would love to know the reasoning behind why/how they send back localtime. I still think it's better to send the actual local time back. If anything, it will provide more accurate hub time keeping?

It does seem that the localtime in the original response from acurite is ahead of actual time, even just by a little bit. Which is causing the daily rain counter to reset before midnight system time. The hub then sends the zeroed out reading which clears the days' counters in the DB. It looks like Acuparse will always have to modify that response. Ensuring that the counters are cleared when Acuparse expects them to be. I'm not totally sure yet how this will affect the readings on myacurite. Since the clock they are using is clearly different. So far, it appears to be reporting OK both within acuparse and myacurite.

Submitting changes to DEV.
